test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,76 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.tag.Tag;
+
+public class UtilsTest {
+    private static final Object NULL_OBJECT = null;
+
+    @Test
+    public void isAnyNull() {
+        ArrayList<String> emptyList = new ArrayList<String>();
+        ArrayList<String> nonEmptyList = new ArrayList<String>();
+        nonEmptyList.add("A string");
```

Comment:
RE why test empty list, if the SUT is likely to have loops, it's good to throw in cases that can push the loop to its boundaries, such as 0 iterations. It's also confirming that given 0 elements the method returns false, which some developers can get wrong. Also, if I'm doing TDD, that would be my first test case (starting with the simplest case). Of course given the simple implementation of the method some of these cases seem redundant, but I prefer test cases to be written in black box fashion to future proof the tests.